templates/aws/base_aws.html

```diff
@@ -4,6 +4,7 @@
   {% block content %}{% endblock %}
 
   <!-- Set default Marketo information for contact form below-->
-  <div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/aws" data-form-id="3790" data-lp-id="" data-return-url="https://ubuntu.com/aws/thank-you" data-lp-url="https://ubuntu.com/aws/contact-us">
-  </div>
+  <div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/form-template" data-form-id="{{ formData.formId }}" data-lp-id="" data-return-url="{{ formData.returnUrl }}" data-lp-url="https://ubuntu.com/aws/contact-us">
```

Comment:
The return URL on successful form submission is https://ubuntu-com-14457.demos.haus/thank-you?referrer=https://ubuntu-com-14457.demos.haus/aws.

Is the referrer URL expected to include the whole URL? I don't have a lot of context on this so feel free to correct me if I'm wrong. Maybe this is something to be updated in the forms template.

Reply:
I had a look at some other forms and that seems to be the pattern we use